### PR TITLE
discord: update to 0.0.11

### PR DESCRIPTION
**Pull request by FirstZero**

### CODE REVIEW

File: srcpkgs/discord/template [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Originally created by Benjamin Hoffmeyer, modified for stable Discord
```
Comment (ericonr) [inline srcpkgs/discord/template:3]: Could you remove this newline and then solve all the resultant xlint warnings? For the license, use `license="custom:Proprietary"`

**Comment by fosslinux** [comment]
Please use the [correct commit naming format](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes).

**Comment by FirstZero** [comment]
> Please use the [correct commit naming format](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes).

Oops, I didnt even realize I didnt use the correct commit message this time. Updated it.

File: srcpkgs/discord/template (hunk 1 above)
Comment (Chocimier) [reply srcpkgs/discord/template:3]: @FirstZero:

> Could you remove this newline and then solve all the resultant xlint warnings?

**Comment by cinerea0** [comment]
@FirstZero you've still got xlint errors. Could you please fix them, run `xlint` on the template, and then repeat until there are no more errors before pushing again?

**Comment by cinerea0** [comment]
I tested it myself, and the following lines for the "top" of the template (everything preceding `do_install`) satisfies xlint:
```
# Template file for 'discord'
pkgname=discord
version=0.0.11
revision=1
archs="x86_64"
wrksrc="Discord"
depends="alsa-lib dbus-glib gtk+3 GConf libnotify nss libXtst libcxx libatomic"
short_desc="Chat and VOIP application"
maintainer="Ryan Conwell <ryanconwell@protonmail.com>"
license="custom:Proprietary"
homepage="https://discordapp.com"
distfiles="https://dl.discordapp.net/apps/linux/${version}/discord-${version}.tar.gz"
checksum=a474761d72ee3fb7fdd31aefc441fc5aa953a2f5dcbe8c30f14fbe105fe458e9
repository=nonfree
restricted=yes
nopie=yes
```

**Comment by FirstZero** [comment]
Im sorry to have left you hanging there, I haven't been home to fix them!
I fixed the linting errors with the latest commit and also took some time to trim down the length of the lines to 80 characters for readability while I was at it.

**Pull request merged**